news/2021-12-08-new-featured-artist-tedjimo-yomigy.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Tedjimo yomigY"
+date: 2021-12-08 09:00:00 +0000
+---
+
+In the mood for an authentic rhythm game sound combined with jazzy bossa nova music? Our newest Featured Artist **Tedjimo yomigY** *somehow* has you covered.
+
+![](https://assets.ppy.sh/artists/223/header.jpg)
+
+**Tedjimo yomigY** incorporates a ton of different inspirations in their one-of-a-kind style, ranging from Nintendo game soundtracks to waltz music to rock 'n' roll. A few of their songs in collaboration with "yoa (dawn-system)" have made their way into other games like *SOUND VOLTEX* and *Muse Dash*, securing **Tedjimo yomigY**'s spot in the minds of rhythm gamers.
+
+Each and every one of the **7** tracks available from [**Tedjimo yomigY**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/223) are masterpieces. Seriously, there's no other way to put it.
+
+Those familiar with the popular beatmap [*Chaos City Niigata - Ukiyoe Yokochou*](https://osu.ppy.sh/beatmapsets/713506) might not know that this track was co-produced by **Tedjimo yomigY**! Check out the map's full difficulty spread through the ultra-wide video below, then experience the rest of these beauties:
```

Comment:
emphasis markers are usually put outside the link if it covers the whole link, but when beatmaps are linked, they don't need any emphasis

```suggestion
Those familiar with the popular beatmap [Chaos City Niigata - Ukiyoe Yokochou](https://osu.ppy.sh/beatmapsets/713506) might not know that this track was co-produced by **Tedjimo yomigY**! Check out the map's full difficulty spread through the ultra-wide video below, then experience the rest of these beauties:
```